tests/utils.py

```diff
@@ -118,3 +135,10 @@ def compose(self, command, cwd):
                 pass
 
         return True
+
+
+class MockUpgrading:
+
+    @staticmethod
+    def migrate_single_to_two_databases(config):
+        pass
```

Comment:
Was this intentionally left for later?

Reply:
It's a mock class only used in tests. 
The idea is just to make it pass over it without doing anything.
We don't need to upgrade the db during the test :-)  
Would you prefer `return` instead of `pass`?

Reply:
`pass` is good 👍 just checking if you wrote the method name and then forgot to complete it 😄